Source/CesiumRuntime/Private/LoadGltfResult.h

```diff
@@ -152,18 +165,45 @@ struct LoadPrimitiveResult {
  * Represents the result of loading a glTF mesh on a game thread.
  */
 struct LoadMeshResult {
+  LoadMeshResult() {}
+
+  LoadMeshResult(const LoadMeshResult&) = delete;
+
+  LoadMeshResult(LoadMeshResult&& other) {
+    primitiveResults.swap(other.primitiveResults);
+  }
+
+  LoadMeshResult& operator=(LoadMeshResult&& other) {
+    primitiveResults.swap(other.primitiveResults);
+    return *this;
+  }
+
   std::vector<LoadPrimitiveResult> primitiveResults{};
 };
 
 /**
  * Represents the result of loading a glTF node on a game thread.
  */
 struct LoadNodeResult {
+  LoadNodeResult() {}
+
+  LoadNodeResult(const LoadNodeResult&) = delete;
+
+  LoadNodeResult(LoadNodeResult&& other)
+      : pInstanceFeatures(std::move(other.pInstanceFeatures)) {
+    InstanceTransforms.swap(other.InstanceTransforms);
+    meshResult.swap(other.meshResult);
+  }
+
   std::optional<LoadMeshResult> meshResult = std::nullopt;
   /**
    * Array of instance transforms, if any.
    */
   std::vector<FTransform> InstanceTransforms;
+  /**
+   * Instance features
```

Comment:
This would benefit from being more descriptive

```suggestion
   * Parses EXT_instance_features from a node. A pointer is used for shared ownership, since there can be multiple meshes and mesh primitives on a node.
```